### PR TITLE
Blazor class libraries topic updates

### LINKED ISSUE #12117: How do consumers of a component utilize the embedded resources or how do I have the component load them?
This page references embedded resources like images and stylesheets but for my sample, the component gets rendered but there are no styles there.  How does this stuff work?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 37b02665-c8ab-bd91-8057-3b07b4e238d3
* Version Independent ID: e4f176f5-0b94-d532-53d5-cffce437961a
* Content: [Razor components class libraries](https://docs.microsoft.com/en-us/aspnet/core/blazor/class-libraries?view=aspnetcore-3.0&tabs=visual-studio#feedback)
* Content Source: [aspnetcore/blazor/class-libraries.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/blazor/class-libraries.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Addresses #10717 
Fixes #12117 

[Internal Review Topic](https://review.docs.microsoft.com/en-us/aspnet/core/blazor/class-libraries?view=aspnetcore-3.0&branch=pr-en-us-12132)

* Overhaul the instructions for set up, including clear VS instructions.
* Enhanced coverage:
  * Razor class library and client-side scenario
  * Static assets and server-side scenario 
    **NOTE**: Do you want to pitch [BlazorEmbedLibrary](https://www.nuget.org/packages/BlazorEmbedLibrary/) as a temp workaround with a support warning? If not, is there any workaround that can be pitched until Preview 7?
* The usual style+grammar pass.
* **QUESTION**: We say static resources are "embedded into the built assembly file," but stylesheets end up in the *_content* folder and referenced from there by *index.html*, so that doesn't seem fully accurate. How should this be addressed?

cc: @4deeptech :rocket:

### CODE REVIEW

**Comment by guardrex** [comment]
@Rick-Anderson I'll make a note for Pre5 on the missed question here on the tracking issue. We can address that later.

File: aspnetcore/blazor/class-libraries.md [hunk 1]
```diff
@@ -20,45 +20,72 @@ Components can be shared in Razor class libraries across projects. Components ca
...
 
-Use the `razorclasslib` (Razor class library) template with the [dotnet new](/dotnet/core/tools/dotnet-new) command:
+## Create a Razor class library
 
-```console
-dotnet new razorclasslib -o MyComponentLib1
-```
+# [Visual Studio](#tab/visual-studio)
+
+To create a Razor class library in Visual Studio:
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/class-libraries.md]: ~To create a Razor class library in Visual Studio:~

File: aspnetcore/blazor/class-libraries.md [hunk 2]
```diff
@@ -20,45 +20,72 @@ Components can be shared in Razor class libraries across projects. Components ca
...
+
+1. Create a new project.
+1. Select **ASP.NET Core Web Application**. Select **Next**.
+1. Provide a project name in the **Project name** field or accept the default project name. The examples in this topic use the project name `MyComponentLib1`. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/class-libraries.md]: Can't you omit location?
~Confirm the **Location** entry is correct or provide a location for the project.~

File: aspnetcore/blazor/class-libraries.md [hunk 3]
```diff
@@ -20,45 +20,72 @@ Components can be shared in Razor class libraries across projects. Components ca
...
+1. In the **Create a new ASP.NET Core Web Application** dialog, confirm that **.NET Core** and **ASP.NET Core 3.0** are selected.
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/class-libraries.md]: Can't you just say"

confirm that ~**.NET Core** and~ **ASP.NET Core 3.0** is selected.

Comment (guardrex) [reply aspnetcore/blazor/class-libraries.md]: Don't think so ... the first dropdown generates the second. If we only say the second, it's not crystal clear where "ASP.NET Core 3.0" comes from. Safest is to say both.

**Comment by guardrex** [comment]
@Rick-Anderson what's up with ...


* ⚠️ aspnetcore/blazor/class-libraries.md （line 25)

   Tab name Visual Studio Code / .NET Core CLI is not approved. Tab names are case-sensitive. See https://aka.ms/ConceptualTabs for details.
* ⚠️ aspnetcore/blazor/class-libraries.md （line 25)

   Tab visual-studio-code+netcore-cli is not approved. Tab IDs are case-sensitive. See https://aka.ms/ConceptualTabs for details.


... it was just passing and AFAIK those are correct.

cc: @scottaddie

**Comment by Rick-Anderson** [comment]
@guardrex  thanks a known false alarm.

**Comment by guardrex** [comment]
I went ahead of the deadline. If that's a problem, plz let me know.

```
# [Visual Studio](#tab/visual-studio)

...

# [Visual Studio Code / .NET Core CLI](#tab/visual-studio-code+netcore-cli)

...

---
```

**Pull request merged**